publication-commons/src/main/java/no/unit/nva/publication/model/business/ImportCandidate.java

```diff
@@ -80,6 +80,18 @@ public Publication toPublication() {
                    .build();
     }
 
+    public ImportCandidate merge(ImportCandidate importCandidate) {
```

Comment:
Dette tenker jeg ikke er bra fra vedlikeholds perspektiv. Hvordan skal vi huske å oppdatere denne funksjonen hvis ImportCandidate får f.eks flere felt?

Gjør det heller på denne måten: ta sortable-identifieren og Impostatus fra den gamle publikasjonen og sett den på den  nye.  Så lagrer du den nye.

Reply:
Da må jeg først slette den gamle, for det ikke er mulig å lagre to ressurser med samme identifier. Kanskje heller bare sette alt bortsett fra importStatus og identifier på den gamle publikasjonen og oppdatere denne.

Reply:
Du kjører en update på den gamle, men du tar den nye og setter identifier fra den gamle posten og kjører update.